source/locale/pt_BR/LC_MESSAGES/internal-communication.po

```diff
@@ -297,107 +576,251 @@ msgid "History"
 msgid "Connecting to the service"
 msgstr "Conectando com o serviço"
 
-#: ../../source/internal-communication.rst:566
-msgid "The connection is done in two steps: you must first obtain a single-use ticket through a HTTP request, then connect to the service via websocket passing it as a parameter."
-msgstr "A conexão é realizada em dois passos, primeiro obtendo um ticket de uso único mediante uma requisição REST, depois conectar ao serviço via websocket passando o ticket como parâmetro."
+#: ../../source/internal-communication.rst:568
+msgid ""
+"The connection is done in two steps: you must first obtain a single-use "
+"ticket through a HTTP request, then connect to the service via websocket "
+"passing it as a parameter."
+msgstr ""
+"A conexão é realizada em dois passos: primeiro é obtido um *ticket* de "
+"uso único via requisição HTTP, e depois o cliente se conecta ao serviço "
+"via websocket passando-o como parâmetro."
 
-#: ../../source/internal-communication.rst:571
+#: ../../source/internal-communication.rst:573
 msgid "First step: Get the single-use ticket"
 msgstr "Primeiro passo: obter um ticket de uso único"
 
-#: ../../source/internal-communication.rst:572
-msgid "A ticket allows the user to subscribe to a dojot topic. To obtain it is necessary to have a JWT access token that is issued by the platform's Authentication/Authorization service. The ticket must be retrieved via a HTTP request using the GET verb to the `<base-url>/kafka-ws/v1/ticket` endpoint. The request must contain the `Authorization` header with the previously retrieved JWT token as a value. Example:"
-msgstr "Um ticket permite o usúario se subscrever em um tópico da dojot. Para obter o ticket é necessário ter um token JWT gerado pelo serviço de Autenticação/Autorização da plataforma. A requisição HTTP para obter um ticket deve ser realizada usando o verbo GET para o endpoint `<base-url>/kafka-ws/v1/ticket`. A requisição deve ter o cabeçalho `Authorização` com o token JWT obtido anteriormente como valor. Exemplo: "
-
-#: ../../source/internal-communication.rst:578
+#: ../../source/internal-communication.rst:574
+msgid ""
+"A ticket allows the user to subscribe to a dojot topic. To obtain it is "
+"necessary to have a JWT access token that is issued by the platform's "
+"Authentication/Authorization service. The ticket must be retrieved via a "
+"HTTP request using the GET verb to the `<base-url>/kafka-ws/v1/ticket` "
+"endpoint. The request must contain the `Authorization` header with the "
+"previously retrieved JWT token as a value. Example:"
+msgstr ""
+"Um ticket permite o usúario se subscrever em um tópico da dojot. Para "
+"obter o ticket é necessário ter um token JWT gerado pelo serviço de "
+"Autenticação/Autorização da plataforma. A requisição HTTP para obter um "
+"ticket deve ser realizada usando o verbo GET para o endpoint `<base-url"
+">/kafka-ws/v1/ticket`. A requisição deve ter o cabeçalho `Authorização` "
```

Comment:
Authorização -> Authorization
I don't think this should be translated, since this is a parameter name in the request.